website/docs/r/vapp_vm.html.markdown

```diff
@@ -140,6 +140,35 @@ resource "vcd_vapp_vm" "internalDiskOverride" {
 
 ```
 
+## Example Usage (Wait for IP addresses on DHCP NIC)
+This example shows how to use [`network_dhcp_wait_seconds`](#network_dhcp_wait_seconds) with DHCP.
+
+```hcl
+resource "vcd_vapp_vm" "TestAccVcdVAppVmDhcpWaitVM" {
+  vapp_name     = vcd_vapp.TestAccVcdVAppVmDhcpWait.name
+  name          = "brr"
+  computer_name = "dhcp-vm"
+  catalog_name  = "cat-dserplis"
+  template_name = "photon-rev2"
+  memory        = 512
+  cpus          = 2
+  cpu_cores     = 1
+
+  network_dhcp_wait_seconds = 220
```

Comment:
No. this is something like a generic thing. Maybe the example deserves a "conservatime" time of 300s (5 minutes)? It may take that time if the image is a beefy one.

Reply:
I concur - better longer timeout here (300s).

Reply:
Adjusted with extra comment to show time value in minutes.